ocpmodels/models/cgcnn.py

```diff
@@ -7,12 +7,13 @@
 
 import torch
 import torch.nn as nn
+from torch_geometric.nn import MessagePassing, global_mean_pool, radius_graph
+from torch_geometric.nn.models.schnet import GaussianSmearing
+
 from ocpmodels.common.registry import registry
 from ocpmodels.common.utils import get_pbc_distances, radius_graph_pbc
-from ocpmodels.datasets.elemental_embeddings import EMBEDDINGS
 from ocpmodels.models.base import BaseModel
-from torch_geometric.nn import MessagePassing, global_mean_pool, radius_graph
-from torch_geometric.nn.models.schnet import GaussianSmearing
+from ocpmodels.models.utils.embeddings import EMBEDDINGS
```

Comment:
I am wondering if we should make part embeddings part of models?
We could consider keeping all embeddings in datasets. So something like ocpmodels.datasets.embeddings.<blah>

Reply:
No preference here, either is fine with me.

Reply:
`ocpmodels.datasets.embeddings` sounds great to me!